classtime.js

```diff
@@ -305,23 +303,29 @@ function isNoSchoolDay() {
 /**
  * This function updates the variables that keep track of the current time and date
  */
-function updateTime() { currentDate = new Date();}
+function updateTime() {
+    if (typeof selectedSchoolIndex !== 'undefined') {
+        currentDate = moment().tz(schools[selectedSchoolIndex].timeZone); 
+    } else {
+        currentDate = moment()
+    }
+}
```

Comment:
the updateTime() method is intended for updating the time in general (i.e. for the local time clock). I think it would be better to use moment to convert to and from GMT when storing and retrieving the times

Reply:
also, if i'm understanding this correctly, I think this might show the local time at the school instead of the users local time if there is a school selected